File: src/main/java/com/example/mcmanager/entity/Comment.java
package com.example.mcmanager.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.io.Serializable;

@TableName(value = "comment")
@Data
public class Comment implements Serializable {
    //主键id
    @TableId(type = IdType.AUTO)
    private Integer comment_id;

    private Integer post_id;
    private Integer userid;
    private String username;
    private String date;
    private String content;
    private String picture;
    private User user;

    @TableField(exist = false)
    private static final long serialVersionUID = 1L;
}

File: vue-admin-template/mock/comment.js
const Mock = require('mockjs')

const data = Mock.mock({
  'items|30': [{
    comment_id: '@id',
    post_id: '@id',
    userid: '@id',
    username: 'name',
    date: '@date',
    content: '@sentence(10, 20)',

  }]
})

module.exports = [
  {
    url: '@api/comment/list',
    type: 'get',
    response: config => {
      const items = data.items
      return {
        code: 20000,
        data: {
          total: items.length,
          items: items
        }
      }
    }
  }
]
